index.js

```diff
@@ -378,11 +378,13 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = ver || msg || opt || undefined
```

Comment:
@bcoe Well it does its job doesn't it? :D I was thinking about that we could maybe create a general global function for this parameter thing though, it would make sense if there are some other functions that need to check first what parameters are provided etc., so we can write code like this:
```javascript
self.version = function (opt, desc, version) {
  var args = getParameters([opt, desc, version], ['opt', 'desc'], ['version']);
  usage.version(args.version)
  self.boolean(args.opt)
  self.describe(args.opt, args.desc)
}
```

The function call passes the parameters, tells which parameters are optional and which required, and then returns an object with the same names or so, was just a random idea though.